scripts/simulation/seat.ts

```diff
@@ -0,0 +1,113 @@
+import { strict as assert } from "assert";
+
+import chalk from "chalk";
+import faker from "faker";
+
+import { SimulatorContext } from "../simulator";
+
+import { takeSeat as actualTakeSeat } from "../lib/bot";
+import { getSectionsRef } from "../lib/collections";
+import { getVenueGridSize } from "../lib/documents";
+import { withErrorReporter } from "../lib/log";
+import { DocumentReference, GridSize } from "../lib/types";
+import { sleep, pickFrom } from "../lib/utils";
+
+export const DEFAULT_SEAT_CHUNK_SIZE = 100;
+export const DEFAULT_SEAT_TICK_MS = 1000;
+export const DEFAULT_SEAT_AFFINITY = 0.01;
+export const DEFAULT_SEAT_IMPATIENCE = 0.01;
+
+export const DEFAULT_GRID_SIZE: GridSize = {
```

Comment:
I have taken care to compartmentalize simulation specific constants into their own files as to not create unnecessary references.